crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -199,7 +202,12 @@ impl TimelineEventKind {
                 Some(AnyMessageLikeEventContent::RoomEncrypted(content)) => {
                     // An event which is still encrypted.
                     if let Some(unable_to_decrypt_info) = unable_to_decrypt_info {
-                        Self::UnableToDecrypt { content, unable_to_decrypt_info }
+                        let utd_cause = UtdCause::determine(
+                            Some(raw_event),
```

Comment:
given that `UtdCause::determine` is only called here (I think?) we could make `raw_event` be not-`Option`

Reply:
Done https://github.com/matrix-org/matrix-rust-sdk/pull/4275/commits/38f1b607d3e07ae1842594e08a415b6976ffb0b5